edgehog/device/forwarder/http.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package edgehog.device.forwarder;
+
+message Http {
+    oneof message {
+        Request request = 1;
+        Response response = 2;
+    }
+
+    message Request {
+        bytes request_id = 1;
+        string path = 2;
+        string method = 3;
+        optional string querystring = 4;
+        map<string, string> headers = 5;
+        optional bytes payload = 6;
```

Comment:
Not sure `payload` is the correct term here. Looking at [some](https://docs.rs/reqwest/latest/reqwest/struct.RequestBuilder.html#method.body) [HTTP](https://requests.readthedocs.io/en/latest/user/advanced/#body-content-workflow) [libraries](https://hexdocs.pm/tesla/Tesla.Env.html#t:body/0) it seems that `body` is the most accurate term for this context

Reply:
Done